.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   root: true,
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint', 'jsdoc', 'jest', 'simple-import-sort'],
+  plugins: ['@typescript-eslint', 'jsdoc', 'jest', 'simple-import-sort', 'import'],
```

Comment:
Nit - can we alphabetically sort these? (Assuming the order `import` is applied doesn't matter with relation to the others).

Reply:
I don't think it should matter, especially since we're only turning on the one rule